### ISSUE TITLE
21-47 Update bricks the C15 running with older EPCs

### ISSUE THREAD

**Issue by AntonSchmied** [nonlinear-labs-dev/C15#2950]
(5i3 and 7i3) if updated over the WebUI. Updating over USB seems to work fine and does not cause any damage.

Observations:
- The update seems to get stuck when updating the BBB (2/3), maybe [here](https://github.com/nonlinear-labs-dev/C15/blob/f509a07701f136ace1bb5eed2d0a43f140043e1c/build-tools/create-c15-update/update_scripts/bbb_update.sh#L88)
- Checksum looks okay (Host PC - ePC - BBB)
- Rights seem to be the same when unpacked  in /update 

Rescue Solution:
- Rescue USB which restores a working OS (21-33 f.e.)

Update Solution:
- Cut WebUI update capability
- Build in a check, to  prevent WebUI update for future updates [run.sh](https://github.com/nonlinear-labs-dev/C15/blob/eea7c7f7216458ed3c87a69017a2d35127dff387/build-tools/create-c15-update/update_scripts/run.sh#L106)

**Comment by majjam** [comment]
I had no issues with 5i3 epc here.  (Was coming from 21-46)
Iused the 21-47 tar from github releases page,
I did the update from webui.

Had issues with the webui before that the tar was not copied to the right  place, but that issue had been closed. (#2852)
Possibly something  related to that ?

**Comment by AntonSchmied** [comment]
> I had no issues with 5i3 epc here. (Was coming from 21-46) Iused the 21-47 tar from github releases page, I did the update from webui.
> 
> Had issues with the webui before that the tar was not copied to the right place, but that issue had been closed. (#2852) Possibly something related to that ?

Thanks for the info!! Q: How did you install the 21-46? USB or WebUI?

**Comment by majjam** [comment]
WebUI

**Comment by majjam** [comment]
if you like i can make an updating test chain with the official update tars.
if you link me the last official tars files i could do that within the next days.

**Comment by AntonSchmied** [comment]
The problem is, the official update tars are the ones, you can find here and cause the same problems.
we will see how far we get today and get back to you! thanks for you support! :)

**Comment by majjam** [comment]
Feel free to contact me if you like and think i can assist.
I know such problems can be really nerve-racking.

**Comment by schlenkibus** [comment]
`systemctl show install-update-from-epc` -> lists all arguments to the service file

```
Dec 01 15:59:34 nonlinux install-update-from-epc.sh[915]: + '[' 0 -eq 3 ']'
Dec 01 15:59:34 nonlinux install-update-from-epc.sh[915]: + systemctl unmask accesspoint
Dec 01 15:59:34 nonlinux systemd[1]: Reloading.
Dec 01 15:59:34 nonlinux systemd-fstab-generator[1212]: Ignoring "noauto" for root device
Dec 01 15:59:34 nonlinux install-update-from-epc.sh[915]: + systemctl enable accesspoint
Dec 01 15:59:34 nonlinux systemd[1]: install-update-from-epc.service: Start operation timed out. Terminating.
Dec 01 15:59:34 nonlinux systemd[1]: Failed to start install-update-from-epc.service.
Dec 01 15:59:34 nonlinux systemd[1]: install-update-from-epc.service: Unit entered failed state.
Dec 01 15:59:34 nonlinux systemd[1]: install-update-rom-epc.service: Failed with result 'timeout'.
```

simple run.sh with endless loop is not being terminated if the update was started via webui, counter was over 700 seconds and no timeout happened. Something is wrong with our assumptions

**Comment by schlenkibus** [comment]
starting an update with 'systemctl start install-update-from-epc-oneshot' seems to go through with the installation. Full log can be found below.

Notice that step 3/3 is not showing up in the log but the final display shows: Updating C15 DONE! Please Restart. Which indicates that the installation was successful. Rebooting the C15 confirms that the installation of plain 21-47 was successful.


```
Dec 02 14:12:23 nonlinux systemd[1]: gettimefromepc.service: Start request repeated too quickly.
Dec 02 14:12:23 nonlinux systemd[1]: Failed to start ePC Time via sshpass.
Dec 02 14:13:12 nonlinux systemd[1]: dev-sda1.device: Job dev-sda1.device/start timed out.
Dec 02 14:13:12 nonlinux systemd[1]: Timed out waiting for device dev-sda1.device.
Dec 02 14:13:12 nonlinux systemd[1]: Dependency failed for /mnt/usb-stick.
Dec 02 14:13:12 nonlinux systemd[1]: mnt-usb\x2dstick.mount: Job mnt-usb\x2dstick.mount/start failed with result 'dependency'.
Dec 02 14:13:12 nonlinux systemd[1]: dev-sda1.device: Job dev-sda1.device/start failed with result 'timeout'.
Dec 02 14:14:13 nonlinux sshd[360]: Connection closed by 192.168.8.4 port 43574 [preauth]
Dec 02 14:14:20 nonlinux sshd[360]: Accepted none for root from 192.168.8.4 port 43612 ssh2
Dec 02 14:14:49 nonlinux sshd[360]: Accepted none for root from 192.168.8.4 port 43792 ssh2
Dec 02 14:15:01 nonlinux sshd[360]: Received disconnect from 192.168.8.4 port 43612:11: disconnected by user
Dec 02 14:15:01 nonlinux sshd[360]: Disconnected from 192.168.8.4 port 43612
Dec 02 14:15:26 nonlinux sshd[360]: Accepted none for root from 192.168.8.4 port 44028 ssh2
Dec 02 14:16:05 nonlinux dhcpd[263]: DHCPREQUEST for 192.168.8.4 from 28:7f:cf:99:23:6b via wlan0
Dec 02 14:16:05 nonlinux dhcpd[263]: DHCPACK on 192.168.8.4 to 28:7f:cf:99:23:6b (justus-TUXEDO) via wlan0
Dec 02 14:16:52 nonlinux systemd[1]: Starting Copy Update From EPC and Execute...
Dec 02 14:16:52 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:16:52 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.11' (ECDSA) to the list of known hosts.
Dec 02 14:16:53 nonlinux sshd[360]: Accepted none for root from 192.168.10.10 port 58784 ssh2
Dec 02 14:17:06 nonlinux sshd[360]: Received disconnect from 192.168.10.10 port 58784:11: disconnected by user
Dec 02 14:17:06 nonlinux sshd[360]: Disconnected from 192.168.10.10 port 58784
Dec 02 14:17:08 nonlinux install-update-from-epc.sh[570]: will update epc
Dec 02 14:17:08 nonlinux install-update-from-epc.sh[570]: will update playcontroller
Dec 02 14:17:08 nonlinux install-update-from-epc.sh[570]: will update bbb
Dec 02 14:17:08 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 1/10
Dec 02 14:17:09 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:09 nonlinux install-update-from-epc.sh[570]: rm: can't remove '/update/EPC/update.tar': No such file or directory
Dec 02 14:17:09 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:10 nonlinux install-update-from-epc.sh[570]: /update/run.sh: line 131: 1: command not found
Dec 02 14:17:10 nonlinux install-update-from-epc.sh[570]: /update/run.sh: line 135: 1: command not found
Dec 02 14:17:10 nonlinux install-update-from-epc.sh[570]:  Starting the C15 update... DO NOT SWITCH OFF! Starting the C15 update... DO NOT SWITCH OFF!
Dec 02 14:17:13 nonlinux install-update-from-epc.sh[570]: Failed to stop playground.service: Unit playground.service not loaded.
Dec 02 14:17:13 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:13 nonlinux systemd[1]: Stopping Nonlinear-Labs BeagleBoneBlackBridge...
Dec 02 14:17:13 nonlinux kernel: lpc_bb_driver_read() : interrupted during read, clearing remaining buffer
Dec 02 14:17:13 nonlinux systemd[1]: Stopped Nonlinear-Labs BeagleBoneBlackBridge.
Dec 02 14:17:13 nonlinux install-update-from-epc.sh[570]:  1/3 Updating... DO NOT SWITCH OFF! 1/3 Updating... DO NOT SWITCH OFF!
Dec 02 14:17:14 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:15 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:18 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:19 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:20 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:20 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:21 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:21 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:22 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:23 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:25 nonlinux install-update-from-epc.sh[570]: Rebooting.
Dec 02 14:17:26 nonlinux kernel: cpsw 4a100000.ethernet eth0: Link is Down
Dec 02 14:17:28 nonlinux kernel: cpsw 4a100000.ethernet eth0: Link is Up - 10Mbps/Full - flow control off
Dec 02 14:17:29 nonlinux install-update-from-epc.sh[570]: Timeout, server 192.168.10.10 not responding.
Dec 02 14:17:29 nonlinux install-update-from-epc.sh[570]: awaiting reboot ... 1/300
Dec 02 14:17:31 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection timed out
Dec 02 14:17:31 nonlinux install-update-from-epc.sh[570]: awaiting reboot ... 2/300
Dec 02 14:17:32 nonlinux kernel: cpsw 4a100000.ethernet eth0: Link is Up - 100Mbps/Full - flow control off
Dec 02 14:17:33 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection timed out
Dec 02 14:17:33 nonlinux install-update-from-epc.sh[570]: awaiting reboot ... 3/300
Dec 02 14:17:35 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection timed out
Dec 02 14:17:35 nonlinux install-update-from-epc.sh[570]: awaiting reboot ... 4/300
Dec 02 14:17:37 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection timed out
Dec 02 14:17:37 nonlinux install-update-from-epc.sh[570]: awaiting reboot ... 5/300
Dec 02 14:17:37 nonlinux kernel: cpsw 4a100000.ethernet eth0: Link is Down
Dec 02 14:17:39 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection timed out
Dec 02 14:17:39 nonlinux install-update-from-epc.sh[570]: awaiting reboot ... 6/300
Dec 02 14:17:40 nonlinux kernel: cpsw 4a100000.ethernet eth0: Link is Up - 100Mbps/Full - flow control rx/tx
Dec 02 14:17:41 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:17:41 nonlinux install-update-from-epc.sh[570]: awaiting reboot ... 7/300
Dec 02 14:17:42 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:17:42 nonlinux install-update-from-epc.sh[570]: awaiting reboot ... 8/300
Dec 02 14:17:43 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:17:43 nonlinux install-update-from-epc.sh[570]: awaiting reboot ... 9/300
Dec 02 14:17:44 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:17:44 nonlinux install-update-from-epc.sh[570]: awaiting reboot ... 10/300
Dec 02 14:17:45 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:17:45 nonlinux install-update-from-epc.sh[570]: awaiting reboot ... 11/300
Dec 02 14:17:46 nonlinux sshd[360]: Accepted none for root from 192.168.10.10 port 37406 ssh2
Dec 02 14:17:46 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:17:46 nonlinux install-update-from-epc.sh[570]: awaiting reboot ... 12/300
Dec 02 14:17:47 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:48 nonlinux install-update-from-epc.sh[570]: ● playground.service - Nonlinear-Labs Playground
Dec 02 14:17:48 nonlinux install-update-from-epc.sh[570]:    Loaded: loaded (/usr/local/lib/systemd/system/playground.service; disabled; vendor preset: disabled)
Dec 02 14:17:48 nonlinux install-update-from-epc.sh[570]:    Active: active (running) since Thu 2021-12-02 14:17:47 UTC; 2s ago
Dec 02 14:17:48 nonlinux install-update-from-epc.sh[570]:  Main PID: 828 (playground)
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]:     Tasks: 17 (limit: 4454)
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]:    Memory: 33.2M
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]:    CGroup: /system.slice/playground.service
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]:            └─828 /usr/local/C15/playground/playground --bbbb-host=192.168.10.11 --midi-bridge=192.168.10.11 --layouts=/usr/local/C15/playground/resources/templates
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]: Dec 02 14:17:47 apl playground[828]: Info:  Successfully set locale to en_US.utf8
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]: Dec 02 14:17:47 apl playground[828]: Debug:  void Settings::load() /sources/projects/epc/playground/src/device-settings/Settings.cpp:181
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]: Dec 02 14:17:47 apl playground[828]: Debug:  void Settings::load() /sources/projects/epc/playground/src/device-settings/Settings.cpp:185
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]: Dec 02 14:17:47 apl playground[828]: Info:  sending setting 31 = 1
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]: Dec 02 14:17:47 apl playground[828]: Warning:  Oled proxy has NO screen set !??
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]: Dec 02 14:17:47 apl playground[828]: Control class ArrowLeft is never used in a layout.
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]: Dec 02 14:17:47 apl playground[828]: Warning:  Oled proxy has NO screen set !??
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]: Dec 02 14:17:47 apl playground[828]: Notify:  Loading presetmanager at /persistent/preset-manager
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]: Dec 02 14:17:47 apl playground[828]: Warning:  Application::Application(int, char**) 112
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]: Dec 02 14:17:47 apl playground[828]: Warning:  void Application::run()
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:49 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:50 nonlinux install-update-from-epc.sh[570]: fsck from util-linux 2.34
Dec 02 14:17:50 nonlinux install-update-from-epc.sh[570]: e2fsck 1.45.3 (14-Jul-2019)
Dec 02 14:17:50 nonlinux install-update-from-epc.sh[570]: Warning!  /dev/sda4 is mounted.
Dec 02 14:17:50 nonlinux install-update-from-epc.sh[570]: Warning: skipping journal recovery because doing a read-only filesystem check.
Dec 02 14:17:50 nonlinux install-update-from-epc.sh[570]: /dev/sda4: clean, 2151/232928 files, 49122/930560 blocks
Dec 02 14:17:50 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:17:50 nonlinux install-update-from-epc.sh[570]: Connection to 192.168.10.10 closed by remote host.
Dec 02 14:17:50 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 1/60
Dec 02 14:17:52 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:17:52 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 2/60
Dec 02 14:17:53 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:17:53 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 3/60
Dec 02 14:17:54 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:17:54 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 4/60
Dec 02 14:17:54 nonlinux sshd[360]: Received disconnect from 192.168.10.10 port 37406:11: disconnected by user
Dec 02 14:17:54 nonlinux sshd[360]: Disconnected from 192.168.10.10 port 37406
Dec 02 14:17:55 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:17:55 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 5/60
Dec 02 14:17:56 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:17:56 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 6/60
Dec 02 14:17:57 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:17:57 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 7/60
Dec 02 14:17:58 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:17:58 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 8/60
Dec 02 14:18:00 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection timed out
Dec 02 14:18:00 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 9/60
Dec 02 14:18:00 nonlinux kernel: cpsw 4a100000.ethernet eth0: Link is Down
Dec 02 14:18:01 nonlinux kernel: cpsw 4a100000.ethernet eth0: Link is Up - 10Mbps/Full - flow control off
Dec 02 14:18:02 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection timed out
Dec 02 14:18:02 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 10/60
Dec 02 14:18:04 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection timed out
Dec 02 14:18:04 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 11/60
Dec 02 14:18:05 nonlinux kernel: cpsw 4a100000.ethernet eth0: Link is Down
Dec 02 14:18:06 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection timed out
Dec 02 14:18:06 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 12/60
Dec 02 14:18:06 nonlinux kernel: cpsw 4a100000.ethernet eth0: Link is Up - 100Mbps/Full - flow control off
Dec 02 14:18:08 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection timed out
Dec 02 14:18:08 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 13/60
Dec 02 14:18:10 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection timed out
Dec 02 14:18:10 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 14/60
Dec 02 14:18:10 nonlinux kernel: cpsw 4a100000.ethernet eth0: Link is Down
Dec 02 14:18:12 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection timed out
Dec 02 14:18:12 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 15/60
Dec 02 14:18:14 nonlinux kernel: cpsw 4a100000.ethernet eth0: Link is Up - 100Mbps/Full - flow control rx/tx
Dec 02 14:18:14 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection timed out
Dec 02 14:18:14 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 16/60
Dec 02 14:18:16 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:18:16 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 17/60
Dec 02 14:18:17 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:18:17 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 18/60
Dec 02 14:18:18 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:18:18 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 19/60
Dec 02 14:18:19 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:18:19 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 20/60
Dec 02 14:18:20 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:18:20 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 21/60
Dec 02 14:18:21 nonlinux install-update-from-epc.sh[570]: ssh: connect to host 192.168.10.10 port 22: Connection refused
Dec 02 14:18:21 nonlinux install-update-from-epc.sh[570]: waiting for OS response ... 22/60
Dec 02 14:18:21 nonlinux sshd[360]: Accepted none for root from 192.168.10.10 port 51168 ssh2
Dec 02 14:18:22 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:18:22 nonlinux systemd[1]: dev-sda1.device: Job dev-sda1.device/start timed out.
Dec 02 14:18:22 nonlinux systemd[1]: Timed out waiting for device dev-sda1.device.
Dec 02 14:18:22 nonlinux systemd[1]: Dependency failed for /mnt/usb-stick.
Dec 02 14:18:22 nonlinux systemd[1]: mnt-usb\x2dstick.mount: Job mnt-usb\x2dstick.mount/start failed with result 'dependency'.
Dec 02 14:18:22 nonlinux systemd[1]: dev-sda1.device: Job dev-sda1.device/start failed with result 'timeout'.
Dec 02 14:18:23 nonlinux kernel: lpc_bb_driver_package_parse() : RX buffer full, dropping packets...
Dec 02 14:18:24 nonlinux install-update-from-epc.sh[570]:  1/3 Updating... DONE! 1/3 Updating... DONE!
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]:  2/3 Updating... DO NOT SWITCH OFF! 2/3 Updating... DO NOT SWITCH OFF!
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]: Warning: Permanently added '192.168.10.10' (ECDSA) to the list of known hosts.
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]: ● accesspoint.service - AccessPoint daemon
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]:    Loaded: loaded (/etc/systemd/system/accesspoint.service; enabled; vendor preset: enabled)
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]:    Active: active (running) since Thu 2016-02-11 16:28:12 UTC; 5 years 9 months ago
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]:  Main PID: 252 (hostapd)
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]:    CGroup: /system.slice/accesspoint.service
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]:            └─252 /usr/sbin/hostapd /etc/hostapd.conf
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]: Dec 02 14:10:49 nonlinux hostapd[252]: wlan0: STA 28:7f:cf:99:23:6b IEEE 802.11: deauthenticated due to local deauth request
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]: Dec 02 14:11:04 nonlinux hostapd[252]: wlan0: STA 28:7f:cf:99:23:6b IEEE 802.11: authenticated
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]: Dec 02 14:11:04 nonlinux hostapd[252]: wlan0: STA 28:7f:cf:99:23:6b IEEE 802.11: authenticated
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]: Dec 02 14:11:04 nonlinux hostapd[252]: wlan0: STA 28:7f:cf:99:23:6b IEEE 802.11: associated (aid 1)
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]: Dec 02 14:11:04 nonlinux hostapd[252]: wlan0: STA 28:7f:cf:99:23:6b IEEE 802.11: associated (aid 1)
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]: Dec 02 14:11:04 nonlinux hostapd[252]: wlan0: AP-STA-CONNECTED 28:7f:cf:99:23:6b
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]: Dec 02 14:11:04 nonlinux hostapd[252]: wlan0: STA 28:7f:cf:99:23:6b RADIUS: starting accounting session FECB9BDE-00000002
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]: Dec 02 14:11:04 nonlinux hostapd[252]: wlan0: STA 28:7f:cf:99:23:6b RADIUS: starting accounting session FECB9BDE-00000002
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]: Dec 02 14:11:04 nonlinux hostapd[252]: wlan0: STA 28:7f:cf:99:23:6b WPA: pairwise key handshake completed (RSN)
Dec 02 14:18:26 nonlinux install-update-from-epc.sh[570]: Dec 02 14:11:04 nonlinux hostapd[252]: wlan0: STA 28:7f:cf:99:23:6b WPA: pairwise key handshake completed (RSN)
Dec 02 14:18:44 nonlinux systemd[1]: Reloading.
Dec 02 14:18:45 nonlinux systemd-fstab-generator[832]: Ignoring "noauto" for root device
Dec 02 14:18:45 nonlinux install-update-from-epc.sh[570]: Created symlink from /etc/systemd/system/multi-user.target.wants/accesspoint.service to /etc/systemd/system/accesspoint.service.
Dec 02 14:18:45 nonlinux systemd[1]: Reloading.
Dec 02 14:18:46 nonlinux systemd-fstab-generator[843]: Ignoring "noauto" for root device
Dec 02 14:18:46 nonlinux systemd[1]: Started AccessPoint daemon.
```

**Comment by AntonSchmied** [comment]
@schlenkibus i think this can be closed, no?
